docs/data/data-grid/tree-data/tree-data.md

```diff
@@ -165,6 +165,38 @@ const invalidRows = [{ path: ['X'] }, { path: ['Y'] }, { path: ['X', 'X'] }];
 
 :::
 
+## Drag-and-drop tree data reordering
+
+With row reordering, users can reorder tree data or move rows from one group to another.
+To enable this feature with row grouping, pass the `rowReordering` prop to the Data Grid component:
+
+```tsx
+<DataGridPro
+  columns={columns}
+  rows={rows}
+  treeData
+  getTreeDataPath={getTreeDataPath}
+  setTreeDataPath={setTreeDataPath}
+  rowReordering
+/>
+```
+
+:::warning
```

Comment:
```suggestion
```

This warning can be removed. To me, those methods are required to make the feature work, it's not a warning.